pyplanet/apps/contrib/voting/__init__.py

```diff
@@ -239,7 +256,8 @@ async def vote_added(self, vote, player):
 			required_votes = (vote.votes_required - len(vote.votes_current))
 			message = '$fff{}$z$s$0cf voted to $fff{}$0cf, $fff{}$0cf more {} needed (use $fffF5$0cf to vote){}.'.format(
 				player.nickname, vote.action, required_votes, ('votes' if required_votes > 1 else 'vote'),
-				(' ($fff{}$0cf seconds left)'.format(int(round((self.current_vote.time_limit - datetime.datetime.now()).total_seconds()))) if self.current_vote.time_limit is not None else '')
+				(' ($fff{}$0cf seconds left)'.format(int(round((
+																	   self.current_vote.time_limit - datetime.datetime.now()).total_seconds()))) if self.current_vote.time_limit is not None else '')
```

Comment:
Why the indenting like this (happens on multiple places)?

Reply:
auto format document ?